frontend/src/css/multiselect.css

```
@@ -1,150 +1,217 @@
-  font-size: 16px;
-  color: #212529;
-}
\ No newline at end of file
+    color: #b2b5b9;
+}
+
+.multi-select .multi-select-header,
+.multi-select .multi-select-option,
+.multi-select .multi-select-all {
+    box-sizing: border-box;
+    display: flex;
```

Stylelint (17.6.0)

[error] Unexpected duplicate "display" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)